rpc/legacyrpc/methods.go

```diff
@@ -3459,14 +3459,14 @@ WrongAddrKind:
 func version(s *Server, icmd interface{}) (interface{}, error) {
 	var resp map[string]dcrjson.VersionResult
 	chainClient, ok := s.requireChainClient()
-	if !ok {
+	if ok {
 		var err error
 		resp, err = chainClient.Version()
 		if err != nil {
 			return nil, err
 		}
 	} else {
-		resp = make(map[string]dcrjson.VersionResult)
+		resp = map[string]dcrjson.VersionResult{}
```

Comment:
Both ways are fine, but I'm curious why you decided to change this

Reply:
I changed it because the map is a reference type and `make` returns a pointer to the type. That'd be a pointer to a reference type so I opted for the reference type.